pdb2pqr/biomolecule.py

```diff
@@ -752,6 +760,9 @@ def apply_name_scheme(self, forcefield_):
                                 # Remove the C/N prefix to keep the protonation state of the residue
                                 # in the terminal residues
                                 rname = rname[1:]
+                            elif rname.startswith("NEUTRAL-"):
+                                # Remove the NEUTRAL-C and NEUTRAL-N prefixes to keep protonation state
+                                rname = rname[9:]
```

Comment:
This is the improvement over my last PR where I was fixing the protonation of terminal residues.